src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -1124,6 +1117,41 @@
       "required": [ "id" ]
     },
 
+    "ruleConfiguration": {
+      "description": "Information about a rule that can be configured at runtime.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+
+        "enabled": {
+          "description": "Specifies whether the rule will be evaluated during the scan.",
+          "type": "boolean"
+        },
+
+        "defaultLevel": {
+          "description": "Specifies the default severity level of the result.",
+          "enum": [ "note", "warning", "error", "open" ]
```

Comment:
The new `ruleConfiguration.defaultLevel` property does have a default value, just as the old `rule.defaultLevel` had. The thing that needs to go away is the default value for the `result.level` property. You can call me if you want to understand why.

Reply:
So you want me to do #105 now?

Reply:
The intent of my comment was that you _not_ remove the default from `ruleConfiguration.defaultLevel`. I forgot that removing the default from `result.level` was a separate issue.

Let's _not_ do #105 yet. If it were just a matter of a one-line change in the schema, I'd say go for it -- but it might have test impact that I'd rather deal with separately.